support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/endpoints/OAuth20UserProfileEndpointController.java

```diff
@@ -67,7 +67,11 @@ public ResponseEntity<String> handlePostRequest(final HttpServletRequest request
     public ResponseEntity<String> handleGetRequest(final HttpServletRequest request,
                                                    final HttpServletResponse response) throws Exception {
         response.setContentType(MediaType.APPLICATION_JSON_VALUE);
-        val accessTokenResult = getAccessTokenFromRequest(request);
+        val accessTokenResult = FunctionUtils.doAndHandle(() -> getAccessTokenFromRequest(request));
+        if (accessTokenResult == null) {
+            LOGGER.error("Unable to get the access token from the request");
+            return buildUnauthorizedResponseEntity(OAuth20Constants.INVALID_REQUEST);
+        }
```

Comment:
Please make sure you have unit tests to catch these lines as well.

Reply:
Yes please., if you don't mind These will show as an "untested" lines without unit tests when we run coverage reports.

Reply:
OK. Will do (tomorrow).

Reply:
I have added a unit test to confirm the behavior.